packages/foam-vscode/src/core/model/workspace.ts

```diff
@@ -119,9 +135,32 @@ export class FoamWorkspace implements IDisposable {
     return identifier;
   }
 
+  /**
+   * Returns a note identifier in reversed order. Used to optimise the storage of notes in
+   * the workspace to optimise retrieval of notes.
+   *
+   * @param reference the URI path to reverse
+   */
+  private getReversedIdentifier(reference: URI | string): string {
```

Comment:
somewhat minor, but I think it makes the code easier to read: can we call this either `getTrieIdentifier`?